File: app/src/main/java/wseemann/media/romote/activity/RemoteActivity.java
package wseemann.media.romote.activity;

import android.os.Bundle;

import wseemann.media.romote.R;

/**
 * Created by wseemann on 6/19/16.
 */
public class RemoteActivity extends ConnectivityActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_remote);
    }
}

File: app/src/main/java/wseemann/media/romote/activity/RemoteActivity.kt
package wseemann.media.romote.activity

import android.os.Bundle
import wseemann.media.romote.R

/**
 * Created by wseemann on 6/19/16.
 */
class RemoteActivity : ConnectivityActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_remote)
    }
}